common/stats/status_server.h

```diff
@@ -35,15 +38,32 @@ class Stats;
 
 class StatusServer {
  public:
+  /*
+   * \brief Endpoint/function name to std::function mapping.
+   *
+   * Stores the mapping of an end-point/function to the the
+   * actual std::function to be executed when requested.
+   *
+   * The std::function takes as input the list of arguments to
+   * the function in the form of a vector of [key, value] pairs
```

Comment:
mentioning that if the endpoint ends with ".txt", the server will return the content generated by op(), otherwise, the server will serialize it to json before returning it?